### ISSUE TITLE
Incorrect use of strict

### ISSUE THREAD

**Issue by spmiller** [Songkeys/tsconfig-to-swcconfig#10]
Hi there,

Many thanks for this library. We are attempting to use it to enable our adoption of swc and it has saved us a lot of time so far!

In [convertTsConfig](https://github.com/Songkeys/tsconfig-to-swcconfig/blob/e5dcbfc5459ebf1a7288d9d6b5266025cdb4c613/src/index.ts#L39), we assign swc config from the tsconfig, setting properties where they don't exist.

The problem occurs when we merge the two configs together [here](https://github.com/Songkeys/tsconfig-to-swcconfig/blob/e5dcbfc5459ebf1a7288d9d6b5266025cdb4c613/src/index.ts#L47). `tsc`'s [definition](https://www.typescriptlang.org/tsconfig#strict) of `strict` is not the same as `swc`'s [definition](https://swc.rs/docs/configuration/modules#strict), but we merge them as if they are the same.

I think a simple solution would be to remove the copying of the `strict` setting from the tsconfig, and only set it if passed in via the manually overridden config. Would you accept a PR that did this?

**Comment by songkeys** [comment]
Hi @spmiller ,

I reconfirmed the docs. You are right! Thank you so much for pointing out.

> Would you accept a PR that did this?

A PR is very welcome!

**Comment by spmiller** [comment]
Hi @Songkeys, thanks for your quick response. I submitted #11 for this, please let me know if you any require further changes.

**Comment by spmiller** [comment]
Hi @Songkeys,

Sorry to bother you, but did the version that was published to NPM as 2.0.0 include this fix? The only change appears to be version number:

```$ diff /tmp/v1/node_modules/tsconfig-to-swcconfig /tmp/v2/node_modules/tsconfig-to-swcconfig
Common subdirectories: /tmp/v1/node_modules/tsconfig-to-swcconfig/dist and /tmp/v2/node_modules/tsconfig-to-swcconfig/dist
diff /tmp/v1/node_modules/tsconfig-to-swcconfig/package.json /tmp/v2/node_modules/tsconfig-to-swcconfig/package.json
3c3
<   "version": "1.5.0",
---
>   "version": "2.0.0",
```

**Comment by songkeys** [comment]
Hi @spmiller

I'm so sorry about this! I mis-published the wrong commit on my local machine. I just published v2.0.1 for the correct one. Thank you so much for pointing out.

I will publish in GitHub Actions in the future!

**Comment by spmiller** [comment]
No problem, these things happen! Thanks for your quick fix 🙂
